cmd/infra/powervs/destroy.go

```diff
@@ -374,8 +389,8 @@ func destroyPowerVsCloudInstance(ctx context.Context, options *DestroyInfraOptio
 	}
 
 	if options.CloudInstanceID != "" {
-		// In case of user provided cloud instance delete only DHCP server
-		err = destroyPowerVsDhcpServer(ctx, infra, cloudInstanceID, session, options.InfraID)
+		log(options.InfraID).Info("Skipping cloud instance deletion because user supplied")
+		return nil
 	} else {
```

Comment:
this cab be outside if/else block because I just this one whole block of code nothing post that..

Reply:
Done.